docs/quickstart/publish_cli.md

```diff
@@ -0,0 +1,122 @@
+description: 'Learn how to publish content with IPFS by pinning a file to a pinning service using a CLI.'
+---
+
+# Publish a file with IPFS
+
+In this quickstart guide, you will learn about [pinning services](../concepts/persistence.md#pinning-in-context) and how to use them to publish content-addressed data with IPFS. To learn the process, you will upload the file to a pinning service called [web3.storage](https://web3.storage/). By the end of this guide, you should have a better understanding of how content addressing and CIDs work from a high level.
+
+:::callout
+The web3.storage pinning service was chosen purely for demonstration purposes, and is one of many [pinning services](../concepts/persistence.md#pinning-in-context) you can choose from. While each pinning services has different SDKs and APIs, their fundamental role is the same - to store files and make them available to the IPFS network. In fact, one of the main benefits of IPFS is that files can be pinned to multiple pinning services, thereby reducing vendor lock-in.
+:::
+
+## Contents <!-- omit from toc -->
+
+- [Overview](#overview)
+- [Pinning services](#pinning-services)
+- [Prerequisites](#prerequisites)
+- [Uploading and pinning a file](#uploading-and-pinning-a-file)
+- [CIDs explained](#cids-explained)
+- [Retrieving with a gateway](#retrieving-with-a-gateway)
+- [Summary and next steps](#summary-and-next-steps)
+
+## Overview
+
+_Pinning_ refers to the process of ensuring that a particular piece of content is retrievable with IPFS. In other words, pinning is equivalent to storing a file on a computer or server that is connected to the internet, thereby making it available to the rest of the IPFS network.
+
+Pinning can be done at various levels, from individual files to entire directories that are addressed by a CID. You can also pin CIDs to multiple IPFS nodes to increase the redundancy and resilience of the file on the network.
+
+## Pinning services
+
+[Pinning services](../concepts/persistence.md#pinning-services) are similar to hosting services, in that they run an IPFS node for you and ensure that your files are available to the IPFS network.
+
+:::callout
+Data pinned to the IPFS network is public by default and retrievable by anyone. Avoid publishing private data or adequately encrypt it before publishing.
+:::
+
+## Prerequisites
+
+- A unix-like terminal
+- Node.js installed to the terminal environment
+- The [following image](../quickstart/images/welcome-to-IPFS.jpg), downloaded and saved on your computer:
+
+![image](../quickstart/images/welcome-to-IPFS.jpg)
+
+## Install and register to w3
+
+1. Install the w3 cli tool.
+   ```
+   npm install -g @web3-storage/w3cli
+   ```
+
+1. Register to w3 using your email.
+
+   ```
+   w3 authorize your@email.com
+   ```
+
+## Upload and pin a file
+
+1. Create a space for your files, and register it.
+
+   ```
+   w3 space create Pictures
+   w3 space register
+   ```
```

Comment:
Break apart steps for readbability, copyability

For some reason, the change suggester isn't formatting this properly. I'll add it manually

Reply:
1. Create a space for your files.

   ```shell
    w3 space create Pictures
   ```
   
1. Register your files.
  
   ```shell
   w3 space register
   ```

Reply:
added in https://github.com/ipfs/ipfs-docs/pull/1548/commits/3c6a6fb5f71bfc5e41f7609bfd5cf73f1cc22367